nuget.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Shouldn't we add `<clear />` before?

Reply:
Hmm, yeah that is what the default output of `dotnet new nugetconfig` looks like.

I removed the `clear`, because we don't really have a problem if our packages are being resolved from other sources, do we? If the Azure Pipelines CI actually is able to resolve a package from its local feed, then that would be fine. LIkewise for our local development, if our packages get resolved from the Visual Studio Offline Fallback Feed, that's okay, no?

It is my understanding that the `clear` element would remove other package sources from the list. And that is not the goal here. The goal here is to explicitly add nuget as a source, to force the CI to consider nuget (because for some unknown reason, it sometimes does not).

Reply:
> I removed the clear, because we don't really have a problem if our packages are being resolved from other sources, do we?

It usually doesn’t matter for restore but it matters when folks install new packages or upgrade all. Many devs have random global feeds which “pollute” the package set/versions which then promptly fail in CI or other people‘s boxes.

In general, I think it’s best practice for projects to have an agreed upon set of feeds, which means the default of `<clear/>` works better.

Does that make sense?

Reply:
That does make sense, I updated and used the output of `dotnet new nugetconfig` verbatim in commit 5c5b49f